tests/test_operator.py

```diff
@@ -640,6 +640,25 @@ def test_tensor(self, func1):
         op2 = Operator([Eq(f, f.dx) for f in f1.values()])
         assert str(op1.ccode) == str(op2.ccode)
 
+    @pytest.mark.parametrize('dtype', [np.complex64, np.complex128])
```

Comment:
Is there a way to not duplicate this test for both CPU and GPU?

Reply:
No the GPU test suite is intentionnally independent and trying to mix things would be too messy